indico/htdocs/sass/modules/event_display/_conferences.scss

```diff
@@ -232,3 +232,40 @@ ul.author-list {
         }
     }
 }
+
+.call-for-abstracts {
```

Comment:
Yeah, I wanted to comment on it as well, I created a new file (`_cfa_page.scss`), @mvidalgarcia you can move your code there.